src/components/icons/MdSmokeFree.tsx

```diff
@@ -0,0 +1 @@
+export { MdSmokeFree} from "react-icons/md";
```

Comment:
would be better to rename file to externalIcons.tsx

export { MdSmokeFree as NoSmokeIcon } from ...